GatinhoInfo/Controllers/TableViewController.swift

```diff
@@ -57,17 +53,13 @@ class TableViewController: UIViewController {
     
     override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
         if let destination = segue.destination as? MoreCatInfoViewController {
-          
             destination.breed = catsInfo[(tableView.indexPathForSelectedRow?.row)!]
```

Comment:
Nós precisamos evitar de usar o force wrap no código, você consegue adicionar aqui uma forma em que não seja necessário forçar (!) esse index?

Isso é importante, pois, caso  o `indexPathForSelectedRow` for nulo, o app vai dar crash.

> Obs.: procure outros locais em que está sendo usado o force wrap no código, para ser removido/evitado o uso dele.

Reply:
Eu imaginei que não teria problema porque se o usuário clicou numa row, ela nunca vai poder ser nula, mas faz sentido se proteger até desse caso

Reply:
A row nunca vai ser nula? 👀 
Só a nível de entendimento: o que acontece no indexPath quando a célula é reciclada?